lib-injection/Dockerfile

```diff
@@ -1,15 +1,33 @@
 # This image provides the files needed to install the ddtrace Python package
 # and auto instrument Python applications in containerized environments.
-FROM busybox
+FROM python:3.10
+WORKDIR /build
+ARG DDTRACE_PYTHON_VERSION
+RUN python3 -m pip install -U pip==23.0.1
+RUN python3 -m pip install packaging==23.0
+RUN mkdir -p pkgs
+ADD ./dl_wheels.py .
+RUN python3 dl_wheels.py \
+        --python-version=3.11 \
+        --python-version=3.10 \
+        --python-version=3.9 \
+        --python-version=3.8 \
+        --python-version=3.7 \
```

Comment:
Is python<3.7 not supported by library injection?

Reply:
Note that it is trivial to add them, they are specifically ignored to keep the package size slim. I will add a comment for this decision in the code too.